### PR TITLE
Add isort & pylint hooks

### PR DESCRIPTION
**Pull request by ls-samer-abuthaher**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -42,11 +42,12 @@ Create a new git tag with the new version and [create a release based on the tag
...
+- `flake8`: Tool For Python Style Guide Enforcement
+- `isort`: Python import sorting
 - `circleci-config-validate`: Test if the CircleCI config file is well formed.
 - `circleci-config-pack`: Pack the CircleCI config and build the config.yml
 - `detect-secrets`: Scan for secrets committed into the repo
 - `end-of-file-fixer`: Checks that all your JSON files are pretty.
-- `flake8`: Tool For Style Guide Enforcement
 - `google-java-code-format`
 - `poetry-pytest`: Use poetry to create a virtual-env and run pytest
 - `pretty-format-json`: Checks that all your JSON files are pretty.
```
Comment (ls-jad-elkik) [inline README.md:53]: They were in alphabetical order!

Comment (ls-samer-abuthaher) [reply README.md:53]: I didnt notice xD I put all python things together

Comment (ls-jad-elkik) [reply README.md:53]: poetry-pytest is still sitting alone 😄

**Comment by ls-jad-elkik** [comment]
We'll need to update https://github.com/lightspeed-hospitality/python-microservice-template/pull/29 (or close it and open a new PR there)

File: .pre-commit-hooks.yaml [hunk 2]
```diff
@@ -6,6 +6,28 @@
...
+  files: \.(py)$
+
+- id: pylint
+  name: pylint
+  description: pylint - It's not just a linter that annoys you!
+  language: python
+  entry: pylint
+  files: \.(py)$
+  require_serial: true
+
```
Comment (ls-jad-elkik) [inline .pre-commit-hooks.yaml:30]: This file is also sorted by id!

**Review by ls-jad-elkik — APPROVED** [review]

**Pull request merged**